lib/ruby_lsp/extension.rb

```diff
@@ -0,0 +1,104 @@
+# typed: strict
+# frozen_string_literal: true
+
+module RubyLsp
+  # To register an extension, inherit from this class and implement both `name` and `activate`
```

Comment:
Should this comment be within `class Extension`?

Reply:
I think the comment needs to be on top for it to be associated with the `Extension` class.